### PR TITLE
Removes dildos and some associated code + minesweeper removal (TESTMERGE PLEASE)

### PR DESCRIPTION
**Pull request by Jaaackx**
## About The Pull Request

entirely removes dildos from code and map. removes them from elsewhere in code very sloppily. also, unrelated, removes minesweeper from BoS base. NO MORE HORSE COCKS, WOOOOOOOOOOOOOOOOOOOO

**need testmerge, this compiled and no obvious runtimes but messing with lewd code is scary as I believe it is pretty engrained in human mobs**

## Why It's Good For The Game

cringe Citadel shit
minesweeper can crash the server

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
del: dildos
del: minesweeper from BoS base
/:cl:

### CODE REVIEW

**Comment by Jaaackx** [comment]
wat why

**Comment by Jaaackx** [comment]
BRO WHY IS THERE A DILDO ON THE DUNGEON Z I FUCKING HATE YOU ALL

File: code/modules/arousal/arousal.dm [hunk 1]
```diff
@@ -246,11 +229,7 @@
 
 	switch(choice)
 		if("Climax alone")
```
Comment (Looreydev) [inline code/modules/arousal/arousal.dm:231]:
```suggestion
```

File: code/modules/arousal/arousal.dm [hunk 2]
```diff
@@ -246,11 +229,7 @@
...
-			if(!available_rosie_palms())
-				return
-			var/obj/item/organ/genital/picked_organ = pick_climax_genitals()
-			if(picked_organ && available_rosie_palms(TRUE))
-				mob_climax_outside(picked_organ)
+			return
```
Comment (Looreydev) [inline code/modules/arousal/arousal.dm:232]:
```suggestion
```

File: code/modules/arousal/arousal.dm [hunk 3]
```diff
@@ -261,17 +240,7 @@
 					if(spillage && in_range(src, partner))
 						mob_climax_partner(picked_organ, partner, spillage == "Yes" ? TRUE : FALSE)
 		if("Fill container")
```
Comment (Looreydev) [inline code/modules/arousal/arousal.dm:242]:
```suggestion
```

File: code/modules/arousal/arousal.dm [hunk 4]
```diff
@@ -261,17 +240,7 @@
...
-				return
-			//We got hands, let's pick an organ
-			var/obj/item/organ/genital/picked_organ
-			picked_organ = pick_climax_genitals() //Gotta be climaxable, not just masturbation, to fill with fluids.
-			if(picked_organ)
-				//Good, got an organ, time to pick a container
-				var/obj/item/reagent_containers/fluid_container = pick_climax_container()
-				if(fluid_container && available_rosie_palms(TRUE, /obj/item/reagent_containers))
-					mob_fill_container(picked_organ, fluid_container)
+			return
```
Comment (Looreydev) [inline code/modules/arousal/arousal.dm:243]:
```suggestion
```

**Review by Looreydev — CHANGES_REQUESTED** [review]

File: code/modules/arousal/arousal.dm [hunk 5]
```diff
@@ -240,43 +223,8 @@
 		return
 
 	//Ok, now we check what they want to do.
-	var/choice = input(src, "Select sexual activity", "Sexual activity:") as null|anything in list("Climax alone","Climax with partner", "Fill container")
+    var/choice = input(src, "Select sexual activity", "Sexual activity:") as null|anything in list("Climax with partner")
```
Comment (Looreydev) [inline code/modules/arousal/arousal.dm:226]:
```suggestion
	var/choice = input(src, "Select sexual activity", "Sexual activity:") as null|anything in list("Climax with partner")
```

**Review by Looreydev — CHANGES_REQUESTED** [review]

**Review by Looreydev — APPROVED** [review]

**Pull request merged**